src/framework/stats.js

```diff
@@ -86,6 +86,4 @@ pc.ApplicationStats = function (device) {
             return pc.Application._currentApplication.batcher._stats;
         }
     });
-
-    pc.events.attach(this);
```

Comment:
We don't seem to add a base class in this case is this intentional?

Reply:
Yes. I can find no evidence that pc.ApplicationStats does anything with events, so maybe this was a copy and paste mistake when the constructor was originally created.